test/lib/Transforms/TestRemappedValue.cpp

```diff
@@ -0,0 +1,90 @@
+//===- TestInlining.cpp - Pass to inline calls in the test dialect --------===//
+//
+// Copyright 2019 The MLIR Authors.
```

Comment:
Can you just insert this in the existing TestPatterns.cpp pass? It's under TestDialect/